Cargo.toml

```
@@ -14,7 +14,7 @@ members = [
 ]
 
 [patch.crates-io]
-halo2_proofs = { git = "https://github.com/privacy-scaling-explorations/halo2.git", tag = "v2022_09_10" }
+halo2_proofs = { path = "../halo2/halo2_proofs" }
```

Do we intend to check in these changes?

```suggestion
halo2_proofs = { git = "https://github.com/privacy-scaling-explorations/halo2.git", tag = "v2022_09_10" }
```

Tip: Edu shared that we can patch in the root Cargo.toml.
```
[patch."https://github.com/privacy-scaling-explorations/halo2.git"]
halo2_proofs = { path = "../halo2/halo2_proofs" }
```